src/data/Title.js

```diff
@@ -0,0 +1,21 @@
+import chipCookie from "@/images/chip.webp";
+import heartCookie from "@/images/heart.webp";
+import swirlCookie from "@/images/swirl.webp";
+
+export const COOKIES = {
+  chip: chipCookie,
+  heart: heartCookie,
+  swirl: swirlCookie,
+};
+
+export const COLORS = {
+  red: {
+    light: "text-red-100",
+    dark: "text-red-200",
+  },
+
+  blue: {
```

Comment:
i think this should be gray, but that's already default

make this purple instead and then add the title for upcoming events